### ISSUE TITLE
Noises are repetitive, why don't take audio that are long enough to not be bored?

### ISSUE THREAD

**Issue by xegulon** [ashutoshgngwr/noice#911]
### Is your feature request related to a problem? Please describe.

Hi, thanks for the amazing and very useful app. I find the shortness of the sounds repeated is quite a problem. Indeed, the fact that a noise is repeated again and again (take as example the birds' song) makes your brain remark it.

### Describe the solution you'd like

Take longer sounds. For example, take an audio of 10min with no repetitions for each kind of noise.

This raises the question of importing its own audio file. There are plenty of good white noises out there in the web, it could be great to be able to mix them with the already existing ones (in the app).

Thanks again!

**Comment by ashutoshgngwr** [comment]
> I find the shortness of the sounds repeated is quite a problem. Indeed, the fact that a noise is repeated again and again (take as example the birds' song) makes your brain remark it.

See https://ashutosh.blog/noice/news/v2-announcement

> This raises the question of importing its own audio file. There are plenty of good white noises out there in the web, it could be great to be able to mix them with the already existing ones (in the app).

See https://github.com/ashutoshgngwr/noice/issues/142

### tldr;

- Longer sounds are coming, albeit they'll be hosted on the internet and most of them will be paid.
- Custom sounds create a plethora of concerns from an user experience perspective. Should we really expect users to create their own perfect loops? There are also issues with Chromecast playback. You can still play custom sounds using another audio player and play Noice alongside it (a supported feature since Noice's initial release).

Feel free to re-open the issue if you want to discuss this further.
